test/features/resource_availability.feature

```diff
@@ -0,0 +1,215 @@
+        Then I fill in "description" with "description"
+        Then I fill in "size" with "1024"
+        Then I press "save"
+        And I wait for 10 seconds
+        Then I should see "resource created by <User> and is available with blank resource_visibility"
+
+        ###
+        # Create resource that NOT available for non-logged in user.
+        ###
+        And I go to "/dataset/new_resource/contains-de-identified-data-no"
+
+        Then I execute the script "document.getElementById('field-image-url').value='http://ckanext-data-qld.docker.amazee.io/'"
+        Then I fill in "name" with "resource created by <User> and is NOT available"
+        Then I fill in "description" with "description"
+        Then I fill in "size" with "1024"
+        Then I select "Resource NOT visible/Pending acknowledgement" from "resource_visibility"
+        Then I press "save"
+        And I wait for 10 seconds
+        Then I should see "resource created by <User> and is NOT available"
+
+        # Verify the result as non-logged in user.
+        Given "Unauthenticated" as the persona
+        And I go to "/dataset/contains-de-identified-data-no"
+        Then I should see "resource created by <User> and is available"
+        Then I should see "resource created by <User> and is available with blank resource_visibility"
+        Then I should not see "resource created by <User> and is NOT available"
+
+        Examples: Users
+            | User          |
+            | Admin         |
+            | TestOrgAdmin  |
+            | TestOrgEditor |
+
+
+    Scenario Outline: An editor, admin or sysadmin user views a dataset resource additional info
+        Given "<User>" as the persona
+        When I log in
+
+        And I go to "/dataset/contains-de-identified-data-yes"
+        Then I press the element with xpath "//a[@title='resource created by Admin and is available']"
+        Then I should see "Re-identification risk governance acknowledgement/Resource visibility"
+
+        Examples: Users
+            | User          |
+            | Admin         |
+            | TestOrgAdmin  |
+            | TestOrgEditor |
+
+
+    Scenario: A Member views a dataset resource additional info
+        Given "TestOrgMember" as the persona
+        When I log in
+
+        And I go to "/dataset/contains-de-identified-data-yes"
+        Then I press the element with xpath "//a[@title='resource created by Admin and is available']"
+        Then I should not see "Re-identification risk governance acknowledgement/Resource visibility"
+
+    Scenario: A general user views a dataset resource additional info
+        Given "Unauthenticated" as the persona
+
+        And I go to "/dataset/contains-de-identified-data-yes"
+        Then I press the element with xpath "//a[@title='resource created by Admin and is available']"
+        Then I should not see "Re-identification risk governance acknowledgement/Resource visibility"
+
+
+    Scenario: Clean up
```

Comment:
Good thinking! Nice way of cleaning up data to re-test locally, never thought about doing that.